docs/python/sdk-guide.md

```diff
@@ -116,7 +116,7 @@ from planet import OrdersClient
 
 async def main():
     async with Session() as sess:
-        client = OrdersClient(sess)
+        client = sess.client('orders')
```

Comment:
with this change, the import of `OrdersClient` on line 115 is no longer necessary